### ISSUE TITLE
Upgrade babel-plugin-emotion's `touch` dependency to ^2.0.1

### ISSUE THREAD

**Issue by mtzhang** [emotion-js/emotion#861]
- `emotion` version: 9.1.2
- `react` version: 16.2.0

**Relevant code.**

https://github.com/emotion-js/emotion/blob/d882e8b71c0fddf12737f9cafeacba2bee4f9278/packages/babel-plugin-emotion/package.json#L24

https://github.com/emotion-js/emotion/blob/d882e8b71c0fddf12737f9cafeacba2bee4f9278/packages/babel-plugin-emotion/src/index.js#L6

https://github.com/emotion-js/emotion/blob/d882e8b71c0fddf12737f9cafeacba2bee4f9278/packages/babel-plugin-emotion/src/index.js#L580

**What you did:**
Within my project makefile

```shell
$ npm install
$ touch node_modules
```

**What happened / Problem description:**
```shell
$ npm install
$ touch node_modules
bad touch!
/Users/mzhang1/Documents/dev/node_modules/touch/bin/touch.js:28
      throw er
      ^

Error: EISDIR: illegal operation on a directory, open 'node_modules'
    at Error (native)
```

I'm using [`react-select`](https://github.com/JedWatson/react-select) in a project, which in turn is using `emotion`. I'm running into a problem with builds because of `emotion`'s outdated dependency (specifically within `babel-package-emotion`) on [`touch`](https://github.com/isaacs/node-touch).

During my project build, the $PATH variable is prepended with `node_modules/.bin`, so local executables can be conveniently used in the Makefile. Because it's prepended, the `touch` dependency takes precedence over the system touch, and because the older version of this library doesn't work with [directories](https://github.com/isaacs/node-touch/issues/8), builds are failing.

**Suggested solution:**

Changing the $PATH to append node_modules/.bin to $PATH, as opposed to prepending it, is not a solution for my project, and so my suggested solution is to 

**update the touch dependency to at least v2.0.1**.  

This is because the name of the CLI installed by the package was changed from `touch.js` to `nodetouch.js` in v2.0.0 ([commit](https://github.com/isaacs/node-touch/commit/8fe84ea23bb1048453b175c59dbc6d2857914b7a)), with a small correction to the bin file name in v2.0.1 ([commit](https://github.com/isaacs/node-touch/compare/v2.0.0...v2.0.1)). This will eliminate the problem of the same-named dependency. 

For additional context, a similar request to upgrade `touch` was made to `nodemon` in July 2017 – see the following issue: https://github.com/remy/nodemon/issues/1072.

**Comment by Andarist** [comment]
Would you like to create a PR for this?

**Comment by mtzhang** [comment]
@Andarist Here it is! https://github.com/emotion-js/emotion/pull/866 

PR template asks me to do the following, but I suspect I do not have permissions to do so:
```
<!-- Please add a `Tag:` prefixed label from the labels so that this PR shows up in the changelog -->
```

**Comment by mtzhang** [comment]
Thanks for @mitchellhamilton for merging the submitted PR! 

Is it possible to cut a patch release with this upgrade soon? Can I get an ETA on that? /cc @Andarist

**Comment by mitchellhamilton** [comment]
Fixed by #866

**Comment by mtzhang** [comment]
@mitchellhamilton Amazing, thank you so much! 🙏
